### ISSUE TITLE
Add ability to "upvote" or "add comment"

### ISSUE THREAD

**Issue by yehosef** [vuejs/vue-hackernews-2.0#170]
I think it would be a helpful example of the Vuex to allow users to be able to "upvote" or "add a comment".  Of course, this is not going to change the backend data and new data would overwrite the changes, but I think it would be an interesting experiment to show how the data is used in the feed and detail view.

**Comment by yyx990803** [comment]
This involves logging into HN on behalf of the user though. AFAIK there's no public API for that.

**Comment by yehosef** [comment]
Yes - so the idea here isn't to do it for real, but to do "as if"..  Meaning allow me to upvote the data or add a comment to the local data, but not try to push it to firebase.. because you can't.  (Or maybe you could write to the local firebase but the sync would fail and ignore the failure.
 But it could be it's not really practical given how firebase works.. I don't know.

What I'm really looking for is something that would work like hackernews - have a feed view, detail view with comments, etc.  And also be able to change them and see the changes reflected in both views (eg, adding a comment would increase the comment count on the detail and list view).  upvoting a post would change the count/hide the upvote on both, etc.

I think it would be valuable to have an example about this level of complexity where the data is mutable. - perhaps there is already something else you could suggest.

Thanks!  Vue is amazing!
